package-overrides/github/sytelus/cryptojs@3.1.2.json

```diff
@@ -0,0 +1,129 @@
+{
+  "main": "core.js",
+  "format": "global",
+  "directories": {
+    "lib": "components"
+  },
+  "shim": {
+    "core.js": {
+      "exports": "CryptoJS"
+    },
+    "sha224.js": {
+      "deps": [
+        "cryptojs/sha256.js"
+      ]
```

Comment:
It uses `CryptoJS` global reference to add a SHA224 item to `CryptoJS.algo`. It doesn't export `CryptoJS.algo.SHA224`; it doesn't even export `CryptoJS` if it's not present.

Requiring `core.js` (via `"*": {"deps": ["cryptojs/core.js"]}`) which exports `CryptoJS` is not enough for SystemJS?